.github/workflows/ci-test.yml

```diff
@@ -488,9 +491,48 @@ jobs:
         with:
           ref: ${{ github.event.pull_request.head.sha }}
           fetch-depth: 0 #get all the history!!!
-      - uses: ./.github/actions/check-machine
+      - uses: ./.github/actions/build-setup
```

Comment:
Is this change related to the switch to `bouncer` + Environment Variables?

Reply:
I'm guessing this is the same answer as here: https://github.com/diem/diem/pull/8133#discussion_r608265014